src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx

```diff
@@ -47,117 +47,6 @@ describe("PluggableAreaChart", () => {
         document.clear();
     });
 
-    it("should reuse one measure, only one category and one category as stack", async () => {
```

Comment:
Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-fe5a846a7ede8c4874d77c3b7706a3ddR236